elements/lisk-chain/src/types.ts

```diff
@@ -40,13 +40,14 @@ export interface AccountJSON {
 	// tslint:disable-next-line readonly-keyword
 	readonly voteWeight: string;
 	readonly nameExist: boolean;
-	readonly multiMin: number;
-	readonly multiLifetime: number;
 	readonly asset: object;
 	// tslint:disable-next-line readonly-keyword
 	readonly votedDelegatesPublicKeys: string[] | null;
-	// tslint:disable-next-line readonly-keyword
-	readonly membersPublicKeys: string[] | null;
+	readonly keys?: {
+		readonly mandatoryKeys?: string[];
```

Comment:
if `keys` exist then doesn't it imply that it will have at least blank object or zero for each of these fields? so we can make these fields required from optional

Reply:
No `keys` could be the default `{}` for accounts that are not multi-signature.